VoodooGPIO/VoodooGPIOIntel.cpp

```diff
@@ -696,13 +699,6 @@ bool VoodooGPIOIntel::start(IOService *provider) {
             continue;
         }
         memset(communities[i].pinInterruptRefcons, 0, sizeof(void *) * communities[i].npins);
-
-        communities[i].isActiveCommunity = IONew(bool, 1);
```

Comment:
Question: Is there any reason as to why we're removing `communities[i].isActiveCommunity = ....`?

Reply:
I didn't see it being used anywhere so I ended up just removing it. The original plan was to just change the type from `bool *` to `bool`.